tailcfg/tailcfg.go

```diff
@@ -94,7 +94,8 @@ type CapabilityVersion int
 //   - 55: 2023-01-23: start of c2n GET+POST /update handler
 //   - 56: 2023-01-24: Client understands CapabilityDebugTSDNSResolution
 //   - 57: 2023-01-25: Client understands CapabilityBindToInterfaceByRoute
-const CurrentCapabilityVersion CapabilityVersion = 57
+//   - 58: 2023-02-08: Client understands CapabilityDebugDisableAlternateDefaultRouteInterface and CapabilityDebugDisableBindConnToInterface
+const CurrentCapabilityVersion CapabilityVersion = 58
```

Comment:
Let's not bump this. Then we can cherry-pick it back. We can't cherry-pick capver bumps.

Reply:
Removed.